service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -893,7 +896,8 @@ internal class RevurderingServiceImpl(
                                 it.id
                             }
                         }.map {
-                            vedtakRepo.lagre(Vedtak.from(it, utbetaling!!.id, clock))
+                            vedtak = Vedtak.from(it, utbetaling!!.id, clock)
+                            vedtakRepo.lagre(vedtak as Vedtak.EndringIYtelse)
```

Comment:
Vedtak.from(it, utbetaling!!.id, clock).also {
vedtak =it
vedtakRepo.lagre(it)
}